frontends/ol-components/src/components/TabButtons/TabButtonList.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import MuiTab from "@mui/material/Tab"
+import type { TabProps } from "@mui/material/Tab"
+import MuiTabList from "@mui/lab/TabList"
+import type { TabListProps } from "@mui/lab/TabList"
+import styled from "@emotion/styled"
+import { Button, ButtonLink } from "../Button/Button"
+import type { ButtonLinkProps, ButtonProps } from "../Button/Button"
+import { css } from "@emotion/react"
+import type { Theme } from "@mui/material/styles"
+
+const defaultTabListProps = {
+  variant: "scrollable",
+  allowScrollButtonsMobile: true,
+  scrollButtons: "auto",
+} as const
+const TabButtonList: React.FC<TabListProps> = styled((props: TabListProps) => (
+  <MuiTabList {...defaultTabListProps} {...props} />
+))([
+  {
+    minHeight: "unset",
+    ".MuiTabs-indicator": {
+      display: "none",
+    },
+    ".MuiTabs-flexContainer": {
+      gap: "8px",
+      alignItems: "center",
+    },
+    ".MuiTabs-scroller": {
+      display: "flex",
+    },
+  },
+])
+
+const tabStyles = ({ theme }: { theme: Theme }) =>
+  css({
+    ":focus-visible": {
+      outlineOffset: "-1px",
```

Comment:
Scrollable tabs include a scroll container with `overflow:hidden` that, without some extra padding, can occlude the browser's focus outline. Rather than adding padding (which would affect layout), I think we should move the focus outline inwards a little bit.